paasta_tools/cli/cmds/local_run.py

```diff
@@ -462,6 +463,30 @@ def add_subparser(subparsers):
         action="store_true",
         default=False,
     )
+    list_parser.add_argument(
+        "--assume-role",
```

Comment:
if you want the ARN, i'd recommend changing this parameter to e.g. `--assume-role-arn` or `--role-arn`. `--assume-role` to me reads as a boolean :)

Reply:
++ agree, and would help remove confusion on just giving role name vs arn